modules/shortlinks.php

```diff
@@ -52,7 +52,7 @@ function wpme_get_shortlink( $id = 0, $context = 'post', $allow_slugs = true ) {
 		if ( empty( $id ) )
 			$id = $blog_id;
 
-		$wpme_url = 'http://wp.me/' . wpme_dec2sixtwo( $id );
+		$wpme_url = 'https://wp.me/' . wpme_dec2sixtwo( $id );
```

Comment:
We can't do this. wp.me will redirect based on the protocol and this makes the assumptions that all sites using it will be https.